src/mscorlib/src/System/Text/ASCIIEncoding.cs

```diff
@@ -78,12 +80,157 @@ public override unsafe int GetByteCount(char* chars, int count)
             return EncodingForwarder.GetByteCount(this, chars, count);
         }
 
-        public override int GetBytes(String chars, int charIndex, int charCount,
-                                              byte[] bytes, int byteIndex)
+        public unsafe override byte[] GetBytes(String s)
+        {
```

Comment:
Looks the changes here is different than previous changes you had before. so we need to evaluate the breaking the changes here. we need to list the scenarios that can be breaking. sub-classing ASCIIEncoding is unlikely but sub-classing UTF8Encoding could be. I'll need some time to look at that.

---
In reply to: [98441711](https://github.com/dotnet/coreclr/pull/9187#discussion_r98441711) [](ancestors = 98441711)